roles/confluent.test/molecule/sharedhost-plaintext-rhel/molecule.yml

```diff
@@ -0,0 +1,104 @@
+  name: docker
+platforms:
+  # Unset env vars resolve to the empty string
+  - name: zookeeper1-kafka-broker1${JOB_BASE_NAME}${BUILD_NUMBER}
+    hostname: zookeeper1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+    groups:
+      - zookeeper
+      - kafka_broker
+    image: geerlingguy/docker-centos7-ansible
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+  - name: schema-registry1${JOB_BASE_NAME}${BUILD_NUMBER}
+    hostname: schema-registry1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+    groups:
+      - schema_registry
+    image: geerlingguy/docker-centos7-ansible
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+  - name: kafka-rest1${JOB_BASE_NAME}${BUILD_NUMBER}
+    hostname: kafka-rest1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+    groups:
+      - kafka_rest
+    image: geerlingguy/docker-centos7-ansible
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+  - name: kafka-connect1${JOB_BASE_NAME}${BUILD_NUMBER}
+    hostname: kafka-connect1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+    groups:
+      - kafka_connect
+    image: geerlingguy/docker-centos7-ansible
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+  - name: ksql1${JOB_BASE_NAME}${BUILD_NUMBER}
+    hostname: ksql1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+    groups:
+      - ksql
+    image: geerlingguy/docker-centos7-ansible
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+  - name: control-center1${JOB_BASE_NAME}${BUILD_NUMBER}
+    hostname: control-center1${JOB_BASE_NAME}${BUILD_NUMBER}.confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+    groups:
+      - control_center
+    image: geerlingguy/docker-centos7-ansible
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    ## Commenting below to avoid port collision on jenkins server
+    ## Uncomment to view c3 in browser at localhost:9021
+    # published_ports:
+    #   - "9021:9021"
+    networks:
+      - name: confluent${JOB_BASE_NAME}${BUILD_NUMBER}
+provisioner:
+  name: ansible
+  config_options:
+    defaults:
+      hash_behaviour: merge
+  playbooks:
+    converge: ../../../../all.yml
+  inventory:
+    group_vars:
+      all:
+        scenario_name: sharedhost-plaintext-rhel
+verifier:
+  name: ansible
+scenario:
+  test_sequence:
+    - lint
```

Comment:
what if you omit the -lint line here and then you wont need all the other changes?

Reply:
Probably not, but It's probably still good to have.

Reply:
nvrm this is incorrect